biggraphite/cli/command_du.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+# Copyright 2016 Criteo
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Du Command."""
+
+from __future__ import print_function
+
+from biggraphite.cli import command
+
+
+_UNITS = ['B', 'KB', 'MB', 'GB', 'TB', 'PB']
```

Comment:
Is there a python lib to do that ? Maybe one that we already use ? Else move it to uitls